dev/msgtest.py

```diff
@@ -3,25 +3,25 @@
 from pprint import pprint
 import sys
 import traceback
+import messagetypes
```

Comment:
Make this into a unit test, maybe add a couple more test cases. This file can be removed completely after the tests are there.